### PR TITLE
Tutorials: add plotly data visualization tutorial from Gábor

### PR DESCRIPTION
**Pull request by bsmth**
This PR adds a tutorial for building a real-time streaming dashboard for market data using Plotly and QuestDB

### CODE REVIEW

File: tutorial/2021-11-01-plotly-finnhub-realtime-dashboard.mdx [hunk 1]
```diff
@@ -0,0 +1,786 @@
...
+
+```python
+finnhub-python==2.4.5  # The official Finnhub Python client
+pydantic==1.8.2        # We will use Pydantic to create data models
+celery[redis]==5.1.2   # Celery will be the periodic task executor
+psycopg2==2.9.1        # We are using QuestDB's PostgreSQL connector
+sqlalchemy==1.4.2      # SQLAlchemy will help us executing SQL queries
+dash==2.0.0            # Dash is used for building data apps
+pandas==1.3.4          # Pandas will handle the data frames from QuestDB
+plotly==5.3.1          # Plotly will help us with beautiful charts
```
Comment (gabor-boros) [inline tutorial/2021-11-01-plotly-finnhub-realtime-dashboard.mdx:162]:
```suggestion
pydantic[dotenv]==1.8.2 # We will use Pydantic to create data models
celery[redis]==5.1.2    # Celery will be the periodic task executor
psycopg2==2.9.1         # We are using QuestDB's PostgreSQL connector
sqlalchemy==1.4.2       # SQLAlchemy will help us executing SQL queries
dash==2.0.0             # Dash is used for building data apps
pandas==1.3.4           # Pandas will handle the data frames from QuestDB
plotly==5.3.1           # Plotly will help us with beautiful charts
```

Comment (gabor-boros) [reply tutorial/2021-11-01-plotly-finnhub-realtime-dashboard.mdx:162]: The requirements should be updated as indicated in [this commit](https://github.com/gabor-boros/questdb-stock-market-dashboard/commit/5cd1b583f3948032fdde290301d3c73781564b05#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)

File: tutorial/2021-11-01-plotly-finnhub-realtime-dashboard.mdx [hunk 2]
```diff
@@ -0,0 +1,786 @@
...
+    query = f"""
+    INSERT INTO quotes(stock_symbol, current_price, high_price, low_price, open_price, percent_change, ts)
+    VALUES(
+        '{symbol}',
+        {quote["c"]},
+        {quote["h"]},
+        {quote["l"]},
+        {quote["o"]},
+        {quote["pc"]},
+        cast({quote["t"]} * 1000000L AS TIMESTAMP)
```
Comment (gabor-boros) [inline tutorial/2021-11-01-plotly-finnhub-realtime-dashboard.mdx:399]:
```suggestion
        systimestamp()
```

Comment (gabor-boros) [reply tutorial/2021-11-01-plotly-finnhub-realtime-dashboard.mdx:399]: Us the current system timestamp over the fetched one

**Review by gabor-boros — COMMENTED** [review]
@bsmth This should be the adjustment we talked about

**Review by marregui — APPROVED** [review]
LGTM

**Pull request merged**